lib/Zonemaster/Backend/Validator.pm

```diff
@@ -211,6 +207,87 @@ sub jsonrpc_method {
     return joi->string->regex( $JSONRPC_METHOD_RE );
 }
 
+sub formats {
+    my ( $rpcapi ) = @_;
+    return {
+        domain => sub { Zonemaster::Backend::Validator::check_domain( $rpcapi, @_ ) },
+        language_tag => sub { Zonemaster::Backend::Validator::check_language_tag( $rpcapi, @_ ) },
+        ip => sub { Zonemaster::Backend::Validator::check_ip( $rpcapi, @_ ) },
```

Comment:
> Since the extra profile validation is removed from the RPCAPI module, but shouldn't it be added here?

Good point, I forgot that one.

> Also, have you considered omitting the Zonemaster::Backend::Validator:: prefixes here?

Done.